telco-core/configuration/reference-crs/required/networking/metallb/metallb.yaml

```diff
@@ -6,5 +6,6 @@ metadata:
   name: metallb
   namespace: metallb-system
 spec:
+  bgpBackend: frr-k8s
```

Comment:
I am torn about setting it explicitly. We changed the default value from 4.16 to 4.17, and openshift users should not change it (unless we face a global disaster that can't be fixed anyway but reverting to the old frr mode).

All in all I'd say users should take the default and stick with it.

Reply:
Agreed, they should not change it.

This reference CR means that Core RDS recommends this option value, even if it is the default value. This can then be used to compare against an arbitrary cluster (customer or not) and flag potential deviations from recommended settings.

@imiller0 please keep me honest here.

Reply:
but this is different, as it will always force the backend to be frr-k8s regardless of any considerations we make in the future (ie we might decide to rollback to frr). The bgpBackend field is not documented and openshift users should never set it in my opinion.

Reply:
I thought it was going to be documented, my bad. If that is not the case, there is no need indeed to mention it in Core RDS. Thanks @fedepaol!